app/views/queue_items/index.html.haml

```diff
@@ -4,29 +4,34 @@
       %article
         %header
           %h2 My Queue
-        %table.table
-          %thead
-            %tr
-              %th(width="10%") List Order
-              %th(width="30%") Video Title
-              %th(width="10%") Play
-              %th(width="20%") Rating
-              %th(width="15%") Genre
-              %th(width="15%") Remove
-          %tbody
-            -@queue_items.each do |queue_item|
+        =form_tag(my_queue_path) do
+          %table.table
+            %thead
               %tr
-                %td
-                  %input.form-control(type="text" value="#{queue_item.order}")
-                %td
-                  = link_to queue_item.video_title, queue_item.video  
-                %td
-                  = button_to "Play", nil, class: "btn btn-default"
-                %td
-                  %span #{queue_item.rating} Stars
-                %td
-                  = link_to queue_item.category_name, queue_item.category
-                %td
-                  = link_to queue_item, method: :delete do
-                    %i.glyphicon.glyphicon-remove
-          = button_to "Update Instant Queue", nil, class: "btn btn-default"
+                %th(width="10%") List Order
+                %th(width="30%") Video Title
+                %th(width="10%") Play
+                %th(width="20%") Rating
+                %th(width="15%") Genre
+                %th(width="15%") Remove
+            %tbody
+              -@queue_items.each do |queue_item|
+                %tr
+                  %td
+                    =hidden_field_tag "queue_items[][id]", queue_item.id
+                    =hidden_field_tag "queue_items[][video_id]", queue_item.video_id
```

Comment:
I don't think passing `video_id` is required. When user is already at this page, all queue items will be created and they already have video_id field. So this would only weaken your security - usually you don't want to expose ids to users or at least in very limited way.